cpp/memilio/mobility/graph.h

```diff
@@ -249,7 +249,7 @@ class Graph
  * @param[in] end_data End date for which the data should be read.
  * @param[in] data_dir Directory that contains the data files.
  * @param[in] population_data_path Path to json file containing the population data.
- * @param[in] node_value Specifies whether the node ids should be district or county ids.
+ * @param[in] node_county Specifies whether the node ids should be county ids (true) or district ids (false).
```

Comment:
why not use `is_county` as above.
I think the naming could be better for a bool variable. (like nodes_are_counties...)